client/my-sites/plans-v2/products-grid-i5/index.tsx

```diff
@@ -70,20 +77,53 @@ const ProductsGridI5: React.FC< ProductsGridProps > = ( {
 		[ duration, availableProducts, includedInPlanProducts, purchasedProducts ]
 	);
 
+	const onResize = useCallback( () => {
```

Comment:
Here we detect when the plan grid is a single row, so that we can apply specific styles.